test_shaders.sh

```diff
@@ -16,4 +16,5 @@ echo "Using spirv-opt in: $(which spirv-opt)."
 ./test_shaders.py shaders-hlsl --hlsl || exit 1
 ./test_shaders.py shaders-hlsl --hlsl --opt || exit 1
 ./test_shaders.py shaders-hlsl-no-opt --hlsl || exit 1
+./test_shaders.py shaders-reflection --reflect || exit 1
```

Comment:
Probably have to change update_shaders.sh as well.

Reply:
done